### DISCUSSION TITLE
How do I change color of sidebar text when I hover over it and show a vertical line?

### DISCUSSION THREAD

**Discussion by howardbaik** [quarto-dev/quarto-cli#9357] [Q&A] [answered]
### Description

In a bookdown based website, I see the sidebar change colors and see a vertical line next to the text. For example, take a look at a recording of this bookdown: https://rstudio4edu.github.io/rstudio4edu-book/edu-stack.html

https://github.com/quarto-dev/quarto-cli/assets/50791792/d2041f1b-83ac-4402-abef-c769b5298a18

I think this chunk of CSS is what's causing this behavior:

```
.summary a:hover {
  color: #68ace5 !important;
}

.book .book-summary ul.summary li.active>a { /*active TOC links*/
  color: #68ace5 !important;
  border-left: solid 4px; /*TODO: This is the vertical line that pops up when you click on it*/
  border-color: #68ace5;
  padding-left: 11px !important;
}
```

How do I replicate this feature in a Quarto Book? Here is the repo that I'm working on: https://github.com/fhdsl/ottr. 

I'm using Quarto version `1.5.29` and macOS Ventura 13.4.1.

Thank you!

**Comment by mcanouil** [comment, marked as answer]
That's a pure HTML/CSS questions.
You need to understand/learn those languages and use your favourite browser inspect/developer mode to see/understand what you want to target.
Once, you get the desired results in your browser, you can insert your (S)CSS in `theme` or in `css` YAML keys. Alternatively, you can use the `include-*` keys.

Beyond that, I'm afraid your question is out of Quarto support scope.

**Reply by howardbaik** [reply]
Oh okay, thanks for letting me know.
